index.js

```diff
@@ -0,0 +1,29 @@
+$(document).ready(function() {
+	$('#myForm').on('submit', function(e) {
+		e.preventDefault();
+        	$("#output").empty();
+        	var searchTerm = $("#searchTerm").val();
+    		var url = 
+		    "https://en.wikipedia.org/w/api.php?action=query&format=json&list=usercontribs&ucuser=" 
+		+ searchTerm + "&ucdir=newer&uclimit=5&callback=?"; 
+    		$.ajax({
+			url: url,
+			type: 'GET',
+			contentType: "application/json; charset=utf-8",
+			async: false,
+       			dataType: "json",
+            		headers: { 'Api-User-Agent': 'useredits-tool/1.0' },
+     			success: function(data, status, jqXHR) {
+     				$("#output").html();
+       				arr = data.query.usercontribs;
+    				console.log(arr);
+    				for(var i=0;i<arr.length;i++) {
+     					$("#output").append(
```

Comment:
In general it's nice to concatenate first and append the whole string as once (less work for the browser, which needs to recompute a bunch of things every time you append something to the DOM). Although it won't make any difference with just five items.

Reply:
Done.